src/olympia/lib/settings_base.py

```diff
@@ -27,6 +27,8 @@
 
 ALLOWED_HOSTS = [
     '.allizom.org',
+    '.amo.nonprod.webservices.mozgcp.net',
+    '.amo.prod.webservices.mozgcp.net',
```

Comment:
I'm aware we don't do this currently, but maybe we should define these per env?  i.e. don't allow the non-prod (`.allizom.org`, `.amo.nonprod.webservices.mozgcp.net`) hosts for the prod environment?

Reply:
Yeah, we should have a follow-up to tighten that up a bit per env.

Reply:
I think that's a good idea in general and worth an issue of its own. @eviljeff would you create one for it?

Meanwhile, I'm going to merge this to get us unblocked on a few tests.

Reply:
#20806